roles/calico/defaults/main.yaml

```diff
@@ -8,3 +8,10 @@ cni_bin_dir: "/opt/cni/bin/"
 calico_etcd_ca_cert_file: "/etc/origin/calico/calico.etcd-ca.crt"
 calico_etcd_cert_file: "/etc/origin/calico/calico.etcd-client.crt"
 calico_etcd_key_file: "/etc/origin/calico/calico.etcd-client.key"
+
+
+calico_ipv6pool_cidr: "fd80:24e2:f998:72d6::/64"
+calico_ipv4pool_ipip: "always"
+calico_ipv4pool_cidr: "192.168.0.0/16"
+
+felix_ipv6support: "false"
```

Comment:
Could be worth extending the main.yml with

```yaml
calico_log_dir: "/var/log/calico"
calico_var_dir: "/var/run/calico"
calico_image: "calico/node:v1.1.0"
modules_dir: "/lib/modules"
```

and updating the `calico.service.j1` file.